quickstart/README.md

```diff
@@ -0,0 +1,54 @@
+# Quarkus demo: Cassandra Client
+
+This example showcases how to use the Cassandra client with Quarkus. 
+
+# Cassandra instance - Running with Docker
+
+Just run it as follows:
+`docker run \
+   --name local-cassandra-instance \
+   -p 7000:7000 \
+   -p 7001:7001 \
+   -p 7199:7199 \
+   -p 9042:9042 \
+   -p 9160:9160 \
+   -p 9404:9404 \
+   -d \
+   launcher.gcr.io/google/cassandra3`
+
+**Provision table**
+
+`docker exec -it local-cassandra-instance cqlsh -e "CREATE KEYSPACE IF NOT EXISTS k1 WITH replication = {'class':'SimpleStrategy', 'replication_factor':1}"`
+`docker exec -it local-cassandra-instance cqlsh -e "CREATE TABLE IF NOT EXISTS k1.fruit(id text, name text, description text, PRIMARY KEY((id), name))"`
```

Comment:
yes, I wanted to make it work this way, but -f file for docker is not working for me.

Reply:
"docker exec" will run the command in the container so the file would need to be available.  I think you're looking for something like this:

```
docker run --name local-cassandra-instance -p 9042:9042 -d -v $PWD/quickstart:/quickstart launcher.gcr.io/google/cassandra3
```

Plus or minus the -v syntax here rather than the newer --mount... but you get the point.  $PWD here is the root of the source checkout.  This should get you to a point where you can do:

```
docker exec -it local-cassandra-instance cqlsh -f /quickstart/setup_db.cql
```

I keep running aground trying that here but I'm trying it on FC28 so selinux is stomping on everything I try to do.  Anyway, this might not be worth the complexity.

Reply:
Yes, mounting the volume only for that purpose is too much.